### ISSUE TITLE
Frida don't support Android Q

### ISSUE THREAD

**Issue by zxhubo** [frida/frida#972]
Hi, my phone os version is Android Q now, when I use frida ,I found Frida don't support Android Q and the error is:
```
Failed to enumerate processes: unable to connect to remote frida-server: closed
```

**Comment by FXTi** [comment]
Have you start fs-server on your phone?

**Comment by zxhubo** [comment]
@FXTi Yes,and my colleagues have the same problem,so I think Frida don't support Andorid Q. do you know Frida can run normally in Android Q or not ?

**Comment by cypherphage** [comment]
https://stackoverflow.com/questions/62171745/frida-server-unable-to-connect?noredirect=1#comment109958929_62171745

**Comment by cypherphage** [comment]
For some reason frida Magisk Module works perfectly fine and runs the server at every boot on android 10.
